mysql-test/suite/rocksdb/t/prefix_extractor_override.test

```diff
@@ -88,6 +83,7 @@ SELECT COUNT(*) FROM t1 WHERE id1=1 AND id2=1 AND id3=1;
 --let $assert_cond="[select variable_value-@u from performance_schema.global_status where variable_name=\'rocksdb_bloom_filter_prefix_checked\']" = 2
 --source include/assert.inc
 
+SET @@global.rocksdb_update_cf_options = '';
```

Comment:
If the final restart is going to be removed, this should be replaced with the usual save/restore idiom